pkgs/by-name/xe/xenia-canary/package.nix

```diff
@@ -42,6 +42,10 @@ llvmPackages_20.stdenv.mkDerivation {
     libuuid
   ];
 
+  postPatch = ''
+    sed -i '/cdialect("C17")/d' premake5.lua
```

Comment:
Can this be replaced with `substituteInPlace`? Reference: https://nixos.org/manual/nixpkgs/unstable/#fun-substituteInPlace

Reply:
Please use `--replace-fail` and it should be good to merge after that, assuming it builds.